### ISSUE TITLE
Weird results on Celeba

### ISSUE THREAD

**Issue by KleinXin** [WendongZh/SPL#5]
I used the mask which hides the center of the image. The pretrained model I used is Ours_celeba.pt.PT.
But the results seems weird.

![image](https://user-images.githubusercontent.com/13084622/134319285-6bf82c16-47a8-4c14-9b23-fda9cc840a6c.png)

**Comment by WendongZh** [comment]
Thanks for your interest.

The provided pre-trained models are all trained with irregular masks provided by [Partial Conv ](https://nv-adlr.github.io/publication/partialconv-inpainting). So our model can not figure out center crop mask. 

I suggest you training our model with center masks to obtain the model you want. It may not take so much time on CelebA dataset.

**Comment by KleinXin** [comment]
> Thanks for your interest.
> 
> The provided pre-trained models are all trained with irregular masks provided by [Partial Conv ](https://nv-adlr.github.io/publication/partialconv-inpainting). So our model can not figure out center crop mask.
> 
> I suggest you training our model with center masks to obtain the model you want. It may not take so much time on CelebA dataset.

Thx! I will try to train the model because I think I can get help from you if I have some problems.
I will also share the pretrained model if its performance is acceptable.

**Comment by jialiang66** [comment]
--mask_index your/npy/file/to/form/img-mask/pairs 

What exactly is added to this place?? I don't quite understand。How did you solve it？
